src/main/java/no/nav/familie/inntektsmelding/server/authz/api/PolicyType.java

```diff
@@ -0,0 +1,7 @@
+package no.nav.familie.inntektsmelding.server.authz.api;
+
+public enum PolicyType {
+    INNTEKTSMELDING,
```

Comment:
Brukes denne for endepunkt som kan kalles av en Arbeidsgiver? 
Det er ikke enkelt med gode enum-navn - INNTEKTSMELDING passer for Resource (hva), mens ARBEIDSGIVER passer for hvem. Oppslag og Innsending er dårlige valg.

Reply:
Ja, det passer kanskje bedre med AG faktisk